app/src/test/java/com/example/whatshouldiread/customclasses/BookTest.java

```diff
@@ -14,4 +15,20 @@ public void setUp() throws Exception {
     @After
     public void tearDown() throws Exception {
     }
+
+    @Test
+    public void bookCons()
```

Comment:
While the assertNotNull _does_ work here as a valid test, it might be better to change it to make sure the default constructor fields are properly being set. @moburke Look into the book class to see what the default constructor sets the various values to.

Reply:
@moburke I created an example test that uses the given, when, then format to test the default constructor for the Question Class. Use that as an example for the book class and to finish the question class.
Commit: 2088c815431abf737433ddb4c82ccba0410d6288